### DISCUSSION TITLE
Keeping checklists organize by providing the ability to sort by alphabetical order.

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2817] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**  

- Yes.

**Is your feature request related to a problem? Please describe.**  

- Yes. When using the **Advanced Checklist [Alpha]** or the **Basic Checklist** note types, the items in said checklist are ordered by their creation, and then once checked off, they are ordered by when they were checked. When it comes to using either of those note with a checklist that should be followed by steps, or a checklist you wish to be able to reuse. It can become quite frustrating.  

**Describe the solution you'd like**

1. Providing the ability to sort the **Advanced Checklist** and **Basic Checklist** by alphabetical order in both the open and closed tasks. That way, if we wished to sort it in such a way. We could format the checklist items as such "01 - This is step one" and "02 - This is step two"
2. _(Not exactly a solution)_ An option to toggle dividing the tasks by open and closed would be a nice option as well. For when you might have to compare a checklist against something. For example, checking an inventory. To make sure you haven't missed an item when you create the checklist.

**Describe alternatives you've considered**

- Nothing comes to mind.

**Additional context**

- No additional context.

**Comment by atmoio** [comment]
We can definitely keep this in mind for the advanced checklist type.

**Comment by diegogurpegui** [comment]
That would be a nice improvement. I would argue that a better improvement is to have the ability to **custom sort the items**, i.e. "drag and drop". That will solve all scenarios.

Compared to alphabetically sorting, drag & drop might be more manual. But for the specific use case of steps list, it won't be much of a problem because in practice one usually put steps in the right order on the first attempts. And eventually one wants to add steps in the middle, which is just a couple of seconds manually.
